Jenkinsfile_nightly

```diff
@@ -16,14 +16,21 @@ def component = "task-configuration-api"
 
 def secrets = [
   'wa-${env}' : [
-    secret('test-law-firm-a-username', 'TEST_LAW_FIRM_A_USERNAME'),
-    secret('test-law-firm-a-password', 'TEST_LAW_FIRM_A_PASSWORD'),
-    secret('idam-redirect-uri', 'IA_IDAM_REDIRECT_URI'),
-    secret('idam-secret', 'IA_IDAM_SECRET'),
-    secret('s2s-name', 'S2S_NAME_WORKFLOW_API')
+  ],
+  's2s-${env}' : [
+    secret('microservicekey-wa-workflow-api', 'S2S_NAME_WORKFLOW_API'),
```

Comment:
this should be the wa-task-configurations-api

Reply:
yeah, I copied and pasted the previous value without thinking twice. Actually, I am going to clean that up since we don't need it here.